apps/web/composables/usePayPal/usePayPal.ts

```diff
@@ -62,6 +56,23 @@ export const usePayPal = () => {
     }
   };
 
+  const updateAvailableAPMs = async (script: PayPalNamespace) => {
+    if (script.getFundingSources && !state.value.activatedAPMs) {
+      state.value.activatedAPMs = true;
+      const availableFoundingSources = new Map();
+      const fundingSources = script.getFundingSources();
+      fundingSources.forEach((fundingSource: string) => {
+        if (script.isFundingEligible) {
+          availableFoundingSources.set(fundingSource, script.isFundingEligible(fundingSource as FUNDING_SOURCE));
+        }
+      });
+      await useSdk().plentysystems.doHandlePayPalPaymentFundingSources({
+        availableFoundingSources: Object.fromEntries(availableFoundingSources),
+      });
+      await usePaymentMethods().fetchPaymentMethods();
```

Comment:
find a way to set a context from where the getScript is loaded so we only fetch the payment methods in checkout, not on product or card